src/pages/expenses/common/hooks.tsx

```diff
@@ -0,0 +1,309 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { Link } from '@invoiceninja/forms';
+import paymentType from 'common/constants/payment-type';
+import { date } from 'common/helpers';
+import { route } from 'common/helpers/route';
+import { useFormatMoney } from 'common/hooks/money/useFormatMoney';
+import { useCurrentCompany } from 'common/hooks/useCurrentCompany';
+import { useCurrentCompanyDateFormats } from 'common/hooks/useCurrentCompanyDateFormats';
+import { useCurrentUser } from 'common/hooks/useCurrentUser';
+import { Expense } from 'common/interfaces/expense';
+import { customField } from 'components/CustomField';
+import { EntityStatus } from 'components/EntityStatus';
+import { StatusBadge } from 'components/StatusBadge';
+import { DataTableColumnsExtended } from 'pages/invoices/common/hooks/useInvoiceColumns';
+import { useTranslation } from 'react-i18next';
+import { ExpenseStatus } from './components/ExpenseStatus';
+
+export const expenseColumns = [
+  'status',
+  'number',
+  'vendor',
+  'client',
+  'date',
+  'amount',
+  'public_notes',
+  'entity_state',
+  'archived_at',
+  //   'assigned_to', @Todo: Need to resolve relationship
+  //   'category', @Todo: Need to resolve relationship
+  'created_at',
+  'created_by',
+  'custom1',
+  'custom2',
+  'custom3',
+  'custom4',
+  'documents',
+  'exchange_rate',
+  'is_deleted',
+  'net_amount', // @Todo: `net_amount` vs `amount`?
```

Comment:
@turbo124 what's diff between `amount` and `net_amount`?

Reply:
net_amount EXCLUDES any taxes